.github/workflows/release.yml

```diff
@@ -468,21 +468,32 @@ jobs:
       run: |
         brew install pandoc
 
-    # The following step can be removed when the build VM is updated with a revision of
-    # vcpkg dating from roughly 01/01/2021 or later. At that point, `doNotUpdateVcpkg`
-    # can be set to `true` and the `vcpkgGitCommitId` can be removed.
-    - name: Update vcpkg
+    - name: Prepare cache key
+      id: key
       run: |
         cd /usr/local/share/vcpkg
-        git fetch --unshallow
+        commitid=$(git show -s --format=%H)
+        echo "::set-output name=commitid::$commitid"
 
-    - name: Prepare vcpkg (with cache)
-      uses: lukka/run-vcpkg@v6
+    - name: Enable vcpkg cache
+      uses: actions/cache@v2
       with:
-        vcpkgDirectory: '/usr/local/share/vcpkg'
-        doNotUpdateVcpkg: false
-        vcpkgGitCommitId: 2a42024b53ebb512fb5dd63c523338bf26c8489c
-        vcpkgArguments: 'liblzma:x64-osx libpng:x64-osx lzo:x64-osx liblzma:arm64-osx libpng:arm64-osx lzo:arm64-osx'
+        path: /usr/local/share/vcpkg/installed
+        key: ${{ runner.os }}-release-vcpkg-${{ steps.key.outputs.commitid }}
+        restore-keys: ${{ runner.os }}-${{ matrix.arch }}-vcpkg-
+
+    - name: Prepare vcpkg
+      run: |
+        vcpkg install \
+          liblzma:x64-osx \
+          liblzma:arm64-osx \
+          libpng:x64-osx \
+          libpng:arm64-osx \
+          lzo:x64-osx \
+          lzo:arm64-osx \
+          zlib:x64-osx \
+          zlib:arm64-osx \
+          #EOF
```

Comment:
The existing instances use `# EOF` (space between `#` and `EOF`). Might be good to be consistent